tools/updateFontAwesome.R

```diff
@@ -100,7 +100,7 @@ v4_icon_df <- data.frame(
 )
 
 # Get info on icons in version 5.
-metadata <- fromJSON(find_package_root_file("data-raw", "fontawesome", "icons.json"))
+metadata <- fromJSON(file.path(source_dir, "metadata", "icons.json"))
 styles <- lapply(metadata, `[[`, "styles")
 
```

Comment:
Near line 167, can we add this line: 

```r
cat('# Generated by tools/updateFontAwesome.R: do not edit by hand\n', file = font_awesome_r)
```

And update the following line to 

```r
cat('font_awesome_brands <- c(\n  "', file = font_awesome_r, append = TRUE)
```

Reply:
Done, thanks for the suggestion!